basic-stashing/README.md

```diff
@@ -0,0 +1,54 @@
+# Git Kata: Basic stashing 
+
+## Setup:
+
+In your terminal, run
+```
+$ ./setup.sh && cd exercise
+```
+
+## The task
+
+You are working on your project. You've staged some work and have some unstaged work as well.
+Suddenly, you're made aware that a bug has made it to production. You'll stash your work, fix the bug and get back to your original work.
+
+1. Explore the repo
+   1. What work do you have in the working directory? 
+   1. What work do you have staged ?  
+   1. What does the commit log look like ?  
+   >*Notice that file.txt has some staged changes (i.e. changes in the index) and unstaged changes (changes in the working directory)* 
+1. Use `git stash` to stash your current work.
+   1. Now, what work do you have in the working directory? 
+   1. What work do you have staged ?
+   1. What does the commit log look like ?
+   1. What does the stash list look like ?
+1. Fix the typos in bug.txt on master and commit your changes.
+1. Now to get back to your work, apply the stash to master.
+   1. What work do you have in the working directory? 
+   1. What work do you have staged ?
+   >*Oops. All our changes are unstaged now. This may be undesirable and unexpected*

```

Comment:
This is very good. Perhaps add a link to the git reset kata, otherwise this is golden :)